FullyNoded/Helpers/Tor/MakeRPCCall.swift

```diff
@@ -183,4 +187,182 @@ class MakeRPCCall {
         }
         
     }
+    
+    // TODO: Move out of here
+    func listUnspentUTXOs(completion: @escaping (Result<[UTXO], MakeRPCCallError>) -> Void) {
+        retry(20, task: { completion in
+            self.executeCommand(method: .listunspent, param: "0", completion: completion)
+        }) { result in
+            switch result {
+            case .success(let data):
+                do {
+                    struct Result: Decodable { // TODO: Move out of here
+                        let utxos: [UTXO]?
+                        let error: Error?
+                        
+                        enum CodingKeys: String, CodingKey {
+                            case utxos = "result"
+                            case error
+                        }
+                        
+                        struct Error: Decodable {
+                            let message: String
+                        }
+                    }
+                    
+                    let decoder = JSONDecoder()
```

Comment:
wouldn't it result in far less code if we decode the `json` inside of `executeNodeCommand` as I currently do? Current method there is only one place we need to decode json from data, with this method there will probably be a couple hundred places.